src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -154,7 +154,12 @@ function(ov_add_onednn)
     endif()
 endfunction()
 
+if(AARCH64 AND (NOT ANDROID))
+    set(ENABLE_SNIPPETS_LIBXSMM_TPP ON)
+endif()
+
 if (ENABLE_SNIPPETS_LIBXSMM_TPP)
+    ov_add_compiler_flags(-Wno-missing-declarations)
```

Comment:
Could you elaborate why you need to add this flag? Can we avoid it?

Reply:
This is to suppress "warning as error" in compile libxsmm. Otherwise there are compile error such as "intel_cpu/thirdparty/libxsmm/src/generator_common_aarch64.c:60:6: error: no previous declaration for ‘libxsmm_generator_vcvt_f32i8_aarch64_sve’ [-Werror=missing-declarations]"

Reply:
Could you please leave a comment that we have to use this flag to avoid thirdparty's compilation errors?

Reply:
comment is added!